swift/src/Ice/Object.swift

```diff
@@ -107,27 +108,3 @@ open class ObjectI<T: SliceTraits>: Object {
         // Do nothing
     }
 }
-
-/// Request dispatcher for plain Object servants.
```

Comment:
With this PR, you can no longer easily instantiate a plain `Ice::Object` servant. I think that's ok.